examples/flask/server.py

```diff
@@ -48,10 +48,66 @@
 import flask
 from flask_cors import CORS
 
-from deepsparse import compile_model
+from deepsparse import Scheduler, compile_model
 from deepsparse.utils import arrays_to_bytes, bytes_to_arrays
 
 
+def engine_flask_server(
+    model_path: str,
+    batch_size: int = 1,
+    num_cores: int = None,
+    num_sockets: int = None,
+    scheduler: Scheduler = None,
+    address: str = "0.0.0.0",
+    port: str = "5543",
+) -> flask.Flask:
+    """
+
+    :param model_path: Either a path to the model's onnx file, a SparseZoo model stub
+        prefixed by 'zoo:', a SparseZoo Model object, or a SparseZoo ONNX File
+        object that defines the neural network
+    :param batch_size: The batch size of the inputs to be used with the model
+    :param num_cores: The number of physical cores to run the model on.
+        Pass None or 0 to run on the max number of cores
+        in one socket for the current machine, default None
+    :param num_sockets: The number of physical sockets to run the model on.
+        Pass None or 0 to run on the max number of sockets for the
+        current machine, default None
+    :param scheduler: The kind of scheduler to execute with. Pass None for the default.
+    :param address: IP address to run on. Default is 0.0.0.0
+    :param port: port to run on. Default is 5543
+    :return: launches a flask server on the given address and port can run the
+        given model on the DeepSparse engine via HTTP requests
+    """
+    print(f"Compiling model at {model_path}")
```

Comment:
Could we potentially change the print statements to logs? Specially for the flask API endpoints.

Reply:
agree with Kevin, would be good to get these moved to logs now and potentially add logging into the classes